toolchain/install/symlink_helpers.bzl

```
@@ -72,6 +80,7 @@ symlink_filegroup = rule(
     implementation = _symlink_filegroup_impl,
     attrs = {
         "out_prefix": attr.string(mandatory = True),
+        "remove_prefix": attr.string(default = ""),
```

Why is the default required here? Isn't where you're adding the setting in `install_filegroups.bzl` the only use?
```suggestion
        "remove_prefix": attr.string(),
```